### PR TITLE
Fixed non copy content of snippet

### PR DESCRIPTION
**Pull request by wachterjohannes**
__tasks:__

- [ ] test coverage
- [ ] gather feedback for my changes

__informations:__

| q                | a
| ---------------- | ---
| Fixed tickets    | none 
| BC breaks        | none 
| Documentation PR | none

### CODE REVIEW

**Comment by danrot** [comment]
changelog is missing :-)

File: src/Sulu/Bundle/ContentBundle/Tests/Functional/Command/ContentLocaleCopyCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+     */
+    private $documentRegistry;
+
+    public function setUp()
+    {
+        $application = new Application($this->getContainer()->get('kernel'));
+        $this->documentManager = $this->getContainer()->get('sulu_document_manager.document_manager');
+        $this->documentRegistry = $this->getContainer()->get('sulu_document_manager.document_registry');
+
+        $cleanupCommand = new ContentLocaleCopyCommand();
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Tests/Functional/Command/ContentLocaleCopyCommandTest.php]: Should probably not be named `$cleanupCommand`.

**Comment by wachterjohannes** [comment]
@danrot fixed everything

File: src/Sulu/Bundle/ContentBundle/Tests/Functional/Command/ContentLocaleCopyCommandTest.php [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+/*
+ * This file is part of the Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Functional\Command;
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Tests/Functional/Command/ContentLocaleCopyCommandTest.php]: wrong namespace

File: tests/Sulu/Component/Content/Compat/Structure/StructureBridgeTest.php [hunk 3]
```diff
@@ -78,4 +81,47 @@ function ($args) use ($title, $images) {
...
+            $metadata->reveal(),
+            $inspector->reveal(),
+            $propertyFactory->reveal(),
+            $document->reveal()
+        );
+
+        $this->assertEquals('test', $structure->getNodeName());
+    }
+
+    public function testGetNodeNameInternal()
```
Comment (danrot) [inline tests/Sulu/Component/Content/Compat/Structure/StructureBridgeTest.php]: I would call this `testGetNodeNameForInternalLink`, just to be 100% clear.

**Comment by danrot** [comment]
Shouldn't we also add a test for the `SnippetController`?

File: src/Sulu/Bundle/SnippetBundle/Tests/Functional/Command/SnippetLocaleCopyCommandTest.php [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+/*
+ * This file is part of the Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Functional\Command;
```
Comment (danrot) [inline src/Sulu/Bundle/SnippetBundle/Tests/Functional/Command/SnippetLocaleCopyCommandTest.php]: wrong namespace

**Comment by wachterjohannes** [comment]
fixed ...

File: src/Sulu/Bundle/SnippetBundle/Tests/Functional/Command/SnippetLocaleCopyCommandTest.php (hunk 4 above)
Comment (danrot) [inline src/Sulu/Bundle/SnippetBundle/Tests/Functional/Command/SnippetLocaleCopyCommandTest.php]: Still the wrong namespace

File: CHANGELOG.md~ [hunk 5]
```diff
@@ -0,0 +1,450 @@
+CHANGELOG for Sulu
```
Comment (danrot) [inline CHANGELOG.md~]: This file should probably be removed

**Comment by danrot** [comment]
I'll fix these two small issues.

**Comment by danrot** [comment]
Can be merged after tests are green.

**Pull request merged**